### PR TITLE
Password reset

### PR DESCRIPTION
**Pull request by rn0**
TODO:
- [x] ~~prevent generating new token on every password reset request~~
- [x] ~~use translations instead of hardcoded messages~~
- [x] ~~add to mail info about user agent and IP which requested password reset~~

### CODE REVIEW

**Comment by rn0** [comment]
@fsi-open

**Comment by JarekW** [comment]
:+1:

**Comment by chives** [comment]
:+1:

File: behat.yml [hunk 1]
```diff
@@ -3,6 +3,8 @@ default:
         default:
             contexts:
                 - FSi\Bundle\AdminSecurityBundle\Behat\Context\AdminUserContext
+                - FSi\Bundle\AdminSecurityBundle\Behat\Context\MailContext:
```
Comment (szymach) [inline behat.yml]: : at the end of class

**Comment by JarekW** [comment]
:+1:

**Comment by szymach** [comment]
:+1:

**Pull request merged**